<testsuite errors="4" failures="1" name="logIn.Login" skipped="0" tests="5" time="1.786766"><testcase classname="logIn.Login" name="Mike clicks login before creating account" status="failed" time="0.801344"><error message="Message: Reached error page: about:neterror?e=dnsNotFound&amp;u=http%3A//django%3A80..." type="WebDriverException">
<![CDATA[
Failing step: Given mike is on nectr site ... failed in 0.797s
Location: features/logIn.feature:10
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "features/steps/logIn.py", line 38, in step_impl
    context.driver.get(context.server_url)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 250, in get
    self.execute(Command.GET, {'url': url})
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 238, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/errorhandler.py", line 193, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=dnsNotFound&u=http%3A//django%3A8000/&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20find%20the%20server%20at%20django.
Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
System info: host: 'df2e6b0362a3', ip: '172.18.0.7', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.57-boot2docker', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=/tmp/rust_mozprofile.7BK0ou7q0Ev4, rotatable=false, timeouts={implicit=0, page load=300000, script=30000}, pageLoadStrategy=normal, platform=ANY, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=52.0.2, platformVersion=4.4.57-boot2docker, moz:processID=5459, browserName=firefox, platformName=linux}]
Session ID: 60d9d42d-c3c3-4863-9931-0755b1601251
Screenshot: available via screen
Stacktrace:
    at sun.reflect.GeneratedConstructorAccessor15.newInstance (None:-1)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:423)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException (W3CHttpResponseCodec.java:133)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:99)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:43)
    at org.openqa.selenium.remote.HttpCommandExecutor.execute (HttpCommandExecutor.java:163)
    at org.openqa.selenium.remote.service.DriverCommandExecutor.execute (DriverCommandExecutor.java:82)
    at org.openqa.selenium.remote.RemoteWebDriver.execute (RemoteWebDriver.java:604)
    at org.openqa.selenium.remote.RemoteWebDriver.get (RemoteWebDriver.java:325)
    at sun.reflect.GeneratedMethodAccessor25.invoke (None:-1)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke (EventFiringWebDriver.java:104)
    at com.sun.proxy.$Proxy6.get (None:-1)
    at org.openqa.selenium.support.events.EventFiringWebDriver.get (EventFiringWebDriver.java:163)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:40)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:25)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)
    at java.lang.Thread.run (Thread.java:745)
]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Scenario: Mike clicks login before creating account
    Given Mike is not registered to nectr ... passed in 0.004s
    Given mike is on nectr site ... failed in 0.797s
    When mike clicks on menu button ... skipped in 0.000s
    And mike clicks on login button on menu ... skipped in 0.000s
    Then mike is redirected to Login page ... skipped in 0.000s
    When mike clicks on "sign_up_redirect" ... skipped in 0.000s
    Then mike is redirected to Signup page ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="logIn.Login" name="Charlie enters correct credentials" status="failed" time="0.320426"><error message="Message: Reached error page: about:neterror?e=dnsNotFound&amp;u=http%3A//django%3A80..." type="WebDriverException">
<![CDATA[
Failing step: Given charlie is on nectr site ... failed in 0.298s
Location: features/logIn.feature:19
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "features/steps/logIn.py", line 38, in step_impl
    context.driver.get(context.server_url)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 250, in get
    self.execute(Command.GET, {'url': url})
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 238, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/errorhandler.py", line 193, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=dnsNotFound&u=http%3A//django%3A8000/&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20find%20the%20server%20at%20django.
Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
System info: host: 'df2e6b0362a3', ip: '172.18.0.7', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.57-boot2docker', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=/tmp/rust_mozprofile.7BK0ou7q0Ev4, rotatable=false, timeouts={implicit=0, page load=300000, script=30000}, pageLoadStrategy=normal, platform=ANY, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=52.0.2, platformVersion=4.4.57-boot2docker, moz:processID=5459, browserName=firefox, platformName=linux}]
Session ID: 60d9d42d-c3c3-4863-9931-0755b1601251
Screenshot: available via screen
Stacktrace:
    at sun.reflect.GeneratedConstructorAccessor15.newInstance (None:-1)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:423)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException (W3CHttpResponseCodec.java:133)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:99)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:43)
    at org.openqa.selenium.remote.HttpCommandExecutor.execute (HttpCommandExecutor.java:163)
    at org.openqa.selenium.remote.service.DriverCommandExecutor.execute (DriverCommandExecutor.java:82)
    at org.openqa.selenium.remote.RemoteWebDriver.execute (RemoteWebDriver.java:604)
    at org.openqa.selenium.remote.RemoteWebDriver.get (RemoteWebDriver.java:325)
    at sun.reflect.GeneratedMethodAccessor25.invoke (None:-1)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke (EventFiringWebDriver.java:104)
    at com.sun.proxy.$Proxy6.get (None:-1)
    at org.openqa.selenium.support.events.EventFiringWebDriver.get (EventFiringWebDriver.java:163)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:40)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:25)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)
    at java.lang.Thread.run (Thread.java:745)
]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Scenario: Charlie enters correct credentials
    Given Charlie is registered to nectr ... passed in 0.022s
    Given charlie is on nectr site ... failed in 0.298s
    When charlie clicks on menu button ... skipped in 0.000s
    And charlie clicks on login ... skipped in 0.000s
    Then charlie is redirected to Login page ... skipped in 0.000s
    When Charlie enters his correct username ... skipped in 0.000s
    And Charlie enters his correct password ... skipped in 0.000s
    And Charlie clicks on sign in ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="logIn.Login" name="Enoc enters incorrect credentials" status="failed" time="0.367232"><error message="Message: Reached error page: about:neterror?e=dnsNotFound&amp;u=http%3A//django%3A80..." type="WebDriverException">
<![CDATA[
Failing step: Given enoc is on nectr site ... failed in 0.360s
Location: features/logIn.feature:30
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "features/steps/logIn.py", line 38, in step_impl
    context.driver.get(context.server_url)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 250, in get
    self.execute(Command.GET, {'url': url})
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 238, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/errorhandler.py", line 193, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=dnsNotFound&u=http%3A//django%3A8000/&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20find%20the%20server%20at%20django.
Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
System info: host: 'df2e6b0362a3', ip: '172.18.0.7', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.57-boot2docker', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=/tmp/rust_mozprofile.7BK0ou7q0Ev4, rotatable=false, timeouts={implicit=0, page load=300000, script=30000}, pageLoadStrategy=normal, platform=ANY, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=52.0.2, platformVersion=4.4.57-boot2docker, moz:processID=5459, browserName=firefox, platformName=linux}]
Session ID: 60d9d42d-c3c3-4863-9931-0755b1601251
Screenshot: available via screen
Stacktrace:
    at sun.reflect.GeneratedConstructorAccessor15.newInstance (None:-1)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:423)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException (W3CHttpResponseCodec.java:133)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:99)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:43)
    at org.openqa.selenium.remote.HttpCommandExecutor.execute (HttpCommandExecutor.java:163)
    at org.openqa.selenium.remote.service.DriverCommandExecutor.execute (DriverCommandExecutor.java:82)
    at org.openqa.selenium.remote.RemoteWebDriver.execute (RemoteWebDriver.java:604)
    at org.openqa.selenium.remote.RemoteWebDriver.get (RemoteWebDriver.java:325)
    at sun.reflect.GeneratedMethodAccessor25.invoke (None:-1)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke (EventFiringWebDriver.java:104)
    at com.sun.proxy.$Proxy6.get (None:-1)
    at org.openqa.selenium.support.events.EventFiringWebDriver.get (EventFiringWebDriver.java:163)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:40)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:25)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)
    at java.lang.Thread.run (Thread.java:745)
]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Scenario: Enoc enters incorrect credentials
    Given Enoc is registered to nectr ... passed in 0.007s
    Given enoc is on nectr site ... failed in 0.360s
    When enoc clicks on menu button ... skipped in 0.000s
    And enoc clicks on nav_Login button ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="logIn.Login" name="Brandon enters correct credentials" status="failed" time="0.29663"><error message="Message: Reached error page: about:neterror?e=dnsNotFound&amp;u=http%3A//django%3A80..." type="WebDriverException">
<![CDATA[
Failing step: And Brandon is on home page of nectr ... failed in 0.292s
Location: features/logIn.feature:41
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "features/steps/logIn.py", line 342, in step_impl
    context.driver.get(context.server_url + "/")
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 250, in get
    self.execute(Command.GET, {'url': url})
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 238, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.5/site-packages/selenium/webdriver/remote/errorhandler.py", line 193, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=dnsNotFound&u=http%3A//django%3A8000/&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20find%20the%20server%20at%20django.
Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
System info: host: 'df2e6b0362a3', ip: '172.18.0.7', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.57-boot2docker', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=/tmp/rust_mozprofile.7BK0ou7q0Ev4, rotatable=false, timeouts={implicit=0, page load=300000, script=30000}, pageLoadStrategy=normal, platform=ANY, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=52.0.2, platformVersion=4.4.57-boot2docker, moz:processID=5459, browserName=firefox, platformName=linux}]
Session ID: 60d9d42d-c3c3-4863-9931-0755b1601251
Screenshot: available via screen
Stacktrace:
    at sun.reflect.GeneratedConstructorAccessor15.newInstance (None:-1)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:423)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException (W3CHttpResponseCodec.java:133)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:99)
    at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode (W3CHttpResponseCodec.java:43)
    at org.openqa.selenium.remote.HttpCommandExecutor.execute (HttpCommandExecutor.java:163)
    at org.openqa.selenium.remote.service.DriverCommandExecutor.execute (DriverCommandExecutor.java:82)
    at org.openqa.selenium.remote.RemoteWebDriver.execute (RemoteWebDriver.java:604)
    at org.openqa.selenium.remote.RemoteWebDriver.get (RemoteWebDriver.java:325)
    at sun.reflect.GeneratedMethodAccessor25.invoke (None:-1)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke (EventFiringWebDriver.java:104)
    at com.sun.proxy.$Proxy6.get (None:-1)
    at org.openqa.selenium.support.events.EventFiringWebDriver.get (EventFiringWebDriver.java:163)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:40)
    at org.openqa.selenium.remote.server.handler.ChangeUrl.call (ChangeUrl.java:25)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)
    at java.lang.Thread.run (Thread.java:745)
]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Scenario: Brandon enters correct credentials
    Given Brandon is registered to nectr and wants to login ... passed in 0.005s
    And Brandon is on home page of nectr ... failed in 0.292s
    When Brandon clicks "Menu" ... skipped in 0.000s
    And Brandon clicks "Log In" button ... skipped in 0.000s
    Then Brandon is redirected to Login page ... skipped in 0.000s
    When Brandon enters "Cashmeousside" in username field on Login page ... skipped in 0.000s
    And Brandon enters "Howboudat" in password field on Login page ... skipped in 0.000s
    And Brandon clicks "Sign In" button ... skipped in 0.000s
    Then Brandon goes to his user profile page, or his "Dashboard" ... undefined in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="logIn.Login" name="Brandon tries to sign in again but enters incorrect credentials" status="failed" time="0.001134"><failure message="" type="AssertionError">
<![CDATA[
Failing step: Given Brandon's registered to nectR with username is "Cashmeousside" ... failed in 0.001s
Location: features/logIn.feature:78
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.5/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "features/steps/logIn.py", line 407, in step_impl
    assert False
AssertionError
]]>
</failure><system-out>
<![CDATA[
@scenario.begin
  Scenario: Brandon tries to sign in again but enters incorrect credentials
    Given Brandon's registered to nectR with username is "Cashmeousside" ... failed in 0.001s
    And Brandon's password is Howboudat ... skipped in 0.000s
    And Brandon is on home page of nectr ... skipped in 0.000s
    And Brandon is not signed in ... skipped in 0.000s
    When Brandon clicks "Menu" ... skipped in 0.000s
    And Brandon clicks "Log In" button ... skipped in 0.000s
    Then Brandon is redirected to "Login" page ... skipped in 0.000s
    And title of the page is "Login" ... skipped in 0.000s
    And page contains an h1 whos text is "Sign In" ... skipped in 0.000s
    When Brandon clicks on username text field ... skipped in 0.000s
    And Brandon enters "Cashmeousside" ... skipped in 0.000s
    And Brandon clicks on password text field ... skipped in 0.000s
    And Brandon enters "Howboudatt" ... skipped in 0.000s
    And Brandon cicks "Sign In" button ... skipped in 0.000s
    Then Brandon is presented with an alert ... skipped in 0.000s
    And username text field is cleared ... skipped in 0.000s
    And password text field is cleared ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase></testsuite>